docs/build.gradle

```diff
@@ -213,6 +213,9 @@ task generateJavaDocs(type: Javadoc) {
     options.links("https://docs.oracle.com/en/java/javase/17/docs/api/")
     options.links("https://docs.opencv.org/4.x/javadoc/")
     options.addStringOption("tag", "pre:a:Pre-Condition")
+    options.addStringOption("Xmaxerrs","9999")
+    options.addStringOption("Xmaxwarns","9999")
+    options.addBooleanOption("Xdoclint/package:-edu.wpi.first.math.proto,-edu.wpi.first.math.controller.*,-edu.wpi.first.math.geometry.*,-edu.wpi.first.math.kinematics.*,-edu.wpi.first.math.system.plant.*,-edu.wpi.first.math.trajectory.proto", true)
```

Comment:
This looks like half of wpimath, and not generated/serialization classes. For instance, PIDController etc would be ignored here, and I suppose we do want it to be checked.

Reply:
`javadoc` says wildcards like that will only count subpackages. I tested to make sure, and I've confirmed it will only exclude the subpackages. Like, everything under `edu.wpi.first.math.geometry` except for the the `proto` and `struct` subpackages will get checked.

Reply:
Still, it's probably more robust to explicitly state the proto and struct subpackages in case we add other subpackages later and don't remember to update this.